app/templates/gentelella/guest/ticketing/pdf/ticket.html

```diff
@@ -28,7 +26,6 @@
 
```

Comment:
@magdalenesuo Should this be also removed, since `number_content` also holds holder info. I'm not sure tho. Can you please check once.